{
	"header": {
		"line": "CVE-2022-3647 (A vulnerability, which was classified as problematic, was found in Red ...)",
		"name": "CVE-2022-3647",
		"description": "(A vulnerability, which was classified as problematic, was found in Red ...)"
	},
	"annotations": [
		{
			"line": "\t- redis <unfixed> (unimportant)",
			"type": "package",
			"package": "redis",
			"kind": "unfixed",
			"flags": [
				{
					"severity": "unimportant"
				}
			]
		},
		{
			"line": "\tNOTE: https://github.com/redis/redis/commit/0bf90d944313919eb8e63d3588bf63a367f020a3",
			"type": "NOTE",
			"description": "https://github.com/redis/redis/commit/0bf90d944313919eb8e63d3588bf63a367f020a3"
		},
		{
			"line": "\tNOTE: Crash inside the crash report when redis already crashed due to calling an invalid",
			"type": "NOTE",
			"description": "Crash inside the crash report when redis already crashed due to calling an invalid"
		},
		{
			"line": "\tNOTE: function pointer, negligible security impact",
			"type": "NOTE",
			"description": "function pointer, negligible security impact"
		}
	]
}
